images/base/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+USER root
+
+RUN microdnf update \
+    && microdnf install java-11-openjdk-headless shadow-utils \
```

```suggestion
    && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install java-11-openjdk-headless shadow-utils \
```

I think this wouldmake the image a bit smaller and with less CVEs.